src/uct/tcp/tcp_ep.c

```diff
@@ -428,14 +437,14 @@ static ucs_status_t uct_tcp_ep_keepalive_enable(uct_tcp_ep_t *ep)
     int intvl_sec;
     ucs_status_t status;
 
-    if ((iface->config.keepalive.idle == 0) ||
-        (iface->config.keepalive.cnt == 0) ||
-        (iface->config.keepalive.intvl == 0)) {
+    if (!uct_tcp_keepalive_is_enabled(iface)) {
         return UCS_OK;
     }
 
-    idle_sec  = ucs_max(1, (int)ucs_time_to_sec(iface->config.keepalive.idle));
-    intvl_sec = ucs_max(1, (int)ucs_time_to_sec(iface->config.keepalive.intvl));
+    idle_sec  = uct_tcp_ep_time_seconds(iface->config.keepalive.idle,
+                                        UCT_TCP_EP_KEEPALIVE_IDLE);
```

Comment:
UCT_TCP_EP_DEFAULT_KEEPALIVE_IDLE

Reply:
renamed to UCT_TCP_EP_DEFAULT_KEEPALIVE_IDLE